src/mplfinance/plotting.py

```diff
@@ -908,10 +908,13 @@ def _addplot_columns(panid,panels,ydata,apdict,xdates,config):
         mark  = apdict['marker']
         color = apdict['color']
         alpha = apdict['alpha']
+        edgecolors  = apdict['edgecolors']
+        linewidths = apdict['linewidths']
+
         if isinstance(mark,(list,tuple,np.ndarray)):
             _mscatter(xdates,ydata,ax=ax,m=mark,s=size,color=color,alpha=alpha)
```

Comment:
Markus-  Do you see any reason not to also add `edgecolors=edgecolors, linewidths=linewidths` to the call to `_mscatter()`.  If you look at the code for `mscatter()` it will simply pass those kwargs down to scatter.

Reply:
Hi @DanielGoldfarb, never thought about it, but of course.